pulse/.swiftlint.yml

```diff
@@ -0,0 +1,38 @@
+disabled_rules:
+- trailing_whitespace
+excluded:
+- Carthage
+- Pods
+- SwiftLint/Common/3rdPartyLib
+line_length:
+    warning: 150
+    error: 200
+    ignores_function_declarations: true
+    ignores_comments: true
+    ignores_urls: true
+force_cast: warning # implicitly. Give warning only for force casting
+force_try:
+  severity: warning
+function_body_length:
+  - 100
+  - 150
+file_length:
+  warning: 400
+  error: 700
+# naming rules can set warnings/errors for min_length and max_length
+# additionally they can set excluded names
+type_name:
+  min_length: 2 # only warning
+  max_length: # warning and error
+    warning: 40
+    error: 50
+  excluded: iPhone # excluded via string
+variable_name:
+  min_length: 2
+  max_length: # warning and error
+    warning: 40
+    error: 50
+cyclomatic_complexity:
+  warning: 15
+  error: 20
+reporter: "xcode" # reporter type (xcode, json, csv, checkstyle)
```

Comment:
Messed up here. I accidentally hard pulled changes from staging into this branch, Wont be a problem when we merge this to staging